plugins/org.polarsys.capella.vp.requirements.ui.commands/src/org/polarsys/capella/vp/requirements/ui/commands/messages.properties

```diff
@@ -3,8 +3,8 @@ GROUP_TITLE_FOR_CONTENT_AREA=Requirement's content
 EXPRESSION_LABEL_TEXT=Expression
 LENGTH_LABEL_TEXT=Length (put nothing to display full text):
 DefaultValueOfLabelExpression=aql:self.ownedAttributes->select( a | a.definition.ReqIFLongName == 'IE PUID').value
-DefaultValueOfContentExpression=aql:OrderedSet{self.ReqIFLongName, self.ReqIFName, self.ReqIFChapterName, self.ReqIFText}->select(s  | s != 'null' and s.size() > 0).prefix('- ')->sep('\n')
+DefaultValueOfContentExpression=aql:OrderedSet{self.ReqIFLongName, self.ReqIFName, self.ReqIFChapterName, self.ReqIFText}->select(s  | s != 'null' and s.size() > 0).prefix('- ')->sep('\\n')

```

Comment:
sure?

Reply:
Yes, this was a subtle bug.
Previously the newline separator worked, **as long as the user did not open the 'Requirement Labels in Diagram' dialog**.

When the user opens the dialog, the string is put into a StringFieldEditor and the newline separator is interpreted. Starting from that moment, the newline separator is ignored in the Diagram Display.

Before:
![before](https://user-images.githubusercontent.com/59692543/105065072-c22be300-5a7d-11eb-8147-1783e498f463.PNG)
![image](https://user-images.githubusercontent.com/59692543/105065225-e982b000-5a7d-11eb-9019-60cf793fcd4c.png)

After:
![after](https://user-images.githubusercontent.com/59692543/105065089-c6f09700-5a7d-11eb-9d03-8906fa07098c.png)
![image](https://user-images.githubusercontent.com/59692543/105065276-f99a8f80-5a7d-11eb-9ac4-b3364ec3630b.png)